library/src/scala/tasty/util/ShowSourceCode.scala

```diff
@@ -721,6 +737,10 @@ class ShowSourceCode[T <: Tasty with Singleton](tasty0: T) extends Show[T](tasty
 
       case Type.ThisType(tp) =>
         printType(tp)
+        tp match {
+          case Type.SymRef(cdef @ ClassDef(_, _, _, _, _), _) if !cdef.flags.isObject => this += ".this"
+          case _ => this
+        }
```

Comment:
Could you explain why special case object here?

Reply:
When refering to the this type of a module we just refere directly to the module itself. For example `scala.Predef` and not `scala.Predef.this`.